deployments/AKS/kubearmor.yaml

```diff
@@ -0,0 +1,1187 @@
+---
```

Comment:
Is this deployment file an exact match of some other deployment file? In that case, can we keep a symbolic link?

Reply:
@nyrahul rather than having a symbolic link, why don't we use deploygen to create the YAML file?